msteams-platform/concepts/design/overview-analytics.md

```diff
@@ -29,18 +29,18 @@ As the app’s developer, you can track your app’s usage in the [Teams app usa
 
 - Monthly, daily, and weekly active users.
 - Retention and intensity charts.
-- Users who have used your app more than five days in the last month.
+- Users who used your app more than five days in the last month.
 - Platform, operating system, and geographic split of users for your app, and so on.
 
 The out-of-box usage reports available in Partner Center or the Developer Portal for Teams can't provide you with in-depth analytics of what goes on inside your app and specific user-level analytics. Such analytics include a user’s journey within your app or a user’s engagement with specific features and measuring scenario completions.
 
-Your app on Teams is essentially a web-based service hosted elsewhere, for example, Azure cloud. It's embedded and surfaced inside Microsoft Teams shell where users can use your app. This applies to your app irrespective of the [platform capabilities](../../overview-explore.md) used, such as tabs, bots, message extensions, meeting extensions, cards, task modules, and so on. All these capabilities are a means to surface web-based experiences inside Teams.
+Your app on Teams is essentially a web-based service hosted elsewhere, for example, Azure cloud. It's embedded and surfaced inside Microsoft Teams shell where users can use your app. This applies to your app irrespective of the [platform capabilities](../../overview-explore.md) used, such as tabs, bots, message extensions, meeting extensions, cards, dialogs (referred as task modules in TeamsJS v.1.0), and so on.. All these capabilities are a means to surface web-based experiences inside Teams.
```

Comment:
How about "TeamsJS v1.x" (to cover all v1 minor versions)

Reply:
Updates